src/jemalloc.c

```diff
@@ -3522,6 +3522,18 @@ je_sdallocx(void *ptr, size_t size, int flags) {
 	LOG("core.sdallocx.exit", "");
 }
 
+JEMALLOC_EXPORT void JEMALLOC_NOTHROW
+je_sdallocx_noflags(void *ptr, size_t size) {
```

Comment:
This isn't part of the public API, and so shouldn't be exported, no? (Or, if not, it should be whitelisted to exempt it from name mangling).

Reply:
that seems right to me

Reply:
Actually, if I am understanding correctly, I think it is part of the public API, just like sdallocx, because the C++ implementation seems to not be part of the library on some (most?) systems.

Reply:
But we only want to expose this to the internal `operator delete` implementation; it's not an ABI we want to commit to.

Reply:
I.e. we don't want to allow raw `je_sdallocx_noflags` calls; the only entry point into this function should be threw `operator delete`.

Reply:
OK, fixed and it passes the CI.